docs/kubernetes.md

```diff
@@ -167,9 +167,150 @@ Max Accuracy: 100.00%
 
 ## TPUs
 
-For TPUs, the same steps for `torchrun` apply: create one job instance per host
-(`--nnodes`) and one process per chip (`--nprocs_per_node`). For more
+Training on TPU is similar to training on GPU in GKE, the same steps for `torchrun` apply. For more
 information about TPU GKE clusters, see [GKE's official
 docs](https://cloud.google.com/kubernetes-engine/docs/how-to/tpus).
 
-TODO: TPU example
+The example below use two ct5lp-hightpu-4t VMs, with 4 v5e TPU each to construct a 2x4 topology nodepool.
+You can adjust the values accordingly to match the training requirement.
+
+Create a new file `tpu_test.yaml` with the following:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: headless-svc
+  namespace: default
+spec:
+  clusterIP: None
+  selector:
+    job-name: torch-xla-tpu-2x4
```

Comment:
Can you change the selector to match on `headless-svc: "true"` like the above example?

Reply:
done